doc/en/pools.rst

```diff
@@ -331,45 +331,72 @@ in modules that matches the name_pattern under current working directory.
 
```

Comment:
Look above, schedule_all() is introduced in the beginning. If you want to introduce the **options param of schedule_all, it goes there. Do more mix schedule_all with make_multitest()

Reply:
then if task object (like `make_multitest`) needs arguments specified by user (not hard coded by `parameters`), how could we pass it ?